_config.yml

```diff
@@ -1,6 +1,6 @@
-highlighter: pygments
 markdown: redcarpet
 redcarpet:
+  syntax_highlighter: rogue
```

Comment:
Has this been tested? This is a typo on `rouge`, and I thought the change needed to be in the top-level `highlighter` field.